models/library.go

```diff
@@ -131,8 +131,12 @@ func getCveContents(cveID string, vul trivyDBTypes.Vulnerability) (contents map[
 
 // FindLockFiles is a list of filenames that is the target of findLock
 var FindLockFiles = []string{
+	// dart/pub
+	ftypes.PubSpecLock,
+	// elixir/mix
+	ftypes.MixLock,
 	// node
-	ftypes.NpmPkgLock, ftypes.YarnLock, ftypes.PnpmLock,
+	ftypes.NpmPkg, ftypes.NpmPkgLock, ftypes.YarnLock, ftypes.PnpmLock,
 	// ruby
 	ftypes.GemfileLock,
```

Comment:
https://aquasecurity.github.io/trivy/v0.48/docs/coverage/language/ruby/
```suggestion
	ftypes.GemfileLock, ".gemspec"
```

Reply:
added at 119b158